instrumentation/p6spy/src/main/java/brave/p6spy/TracingJdbcEventListener.java

```diff
@@ -30,7 +30,7 @@
 final class TracingJdbcEventListener extends SimpleJdbcEventListener {
 
   private static final Pattern URL_SERVICE_NAME_FINDER =
-    Pattern.compile("zipkinServiceName=(\\w*)");
+    Pattern.compile("zipkinServiceName=(.*?)(?:&|$)");
```

Comment:
Could you maybe add a small comment what the regex is aiming to match ? Not everyone can work it out from the top of their head.

Reply:
Comment included.